src/lambda/explorer/cloudFormationNodes.ts

```diff
@@ -63,6 +63,15 @@ export class DefaultCloudFormationNode extends AWSTreeErrorHandlerNode implement
 
         return !!this.errorNode ? [this.errorNode]
             : [...this.stackNodes.values()]
+                .sort((nodeA, nodeB) =>
+                    nodeA.stackName.localeCompare(
+                        nodeB.stackName,
+                        undefined,
+                        {
+                            sensitivity: 'base'
```

Comment:
What's the reason for using `base` over the default `variant`?

Reply:
I now think I should leave this as the default `variant` (case/accent sensitive), to respect situations where users have a naming convention that uses case.